src/picker.rs

```diff
@@ -406,11 +501,13 @@ impl Parser {
             }
             ParsedResponse::Sixel(_) => match next {
                 'c' => {
+                    self.data.push(next);
+
                     // This is just easier than actually parsing the string.
                     let is_sixel = self.data.contains(";4;")
                         || self.data.contains("?4;")
-                        || self.data.contains(";4")
-                        || self.data.contains("?4");
+                        || self.data.contains(";4c")
+                        || self.data.contains("?4c");
```

Comment:
FYI, you shouldn't need to check the variants beginning with `?`. If the first parameter of the `DA` response is a `4`, that indicates that the terminal is a VT132 terminal, which did _not_ support sixel.

Technically the first parameter should be 60 or more if you follow the spec exactly, but there are a number of modern terminals that aren't spec compliant, so you're probably better off ignoring that particular requirement.

In short, I'd recommend checking from parameter 2 onwards, i.e. you only need to search for `";4;"` or `";4c"`.

Reply:
Thanks for the comment, I learned something new.
But here is just fixing a previous bug, I think simplifying this check is another issue that should be put into a new PR to fix.